Meeting minutes 2025-10-02 (#25)
solid-contrib/practitioners: meetings/2025-10-02.md

- chat https://matrix.to/#/#solid-practitioners:matrix.org
- repo https://github.com/solid-contrib/practitioners
- recordings https://spectra.video/c/solid_practitioners/videos

## Facilitator

- Michal

## Scribe

- Theo

## Participants (please add yourself)

- Jeff Zucker
- [Luke Dary](https://www.linkedin.com/in/lukedary)
- [Michal](https://id.mrkvon.org)
- Tom Byrd
- [Fred Gibson](https://graphmetrix.com)
- [Theo @thhck](http://github.com/thhck)
- [Rui Zhao](https://me.ryey.icu)
- Marc Haddle
- [Jesse Wright](https://jeswr.solidcommunity.net/)
- [Wilfred Pinfold](https://www.linkedin.com/in/wilfred-pinfold-aa6b68/)
- John Kirkwood
- \_

## Agenda

### Check-in

- How are you coming to the meeting? (brief, round, 3-5 min, not minuted)

### Administrative

- Call for Scribe
- ~~Objections to recording session?~~ (not recorded)
- Review agenda

### New Member Check-in (brief intro, current work)

- Luke: work at Redhat. Here to talk about adoption of Solid. Also volunteer with non-profit, which pay money for auth or storage, would be a lower cost for them to use Solid.
- ?: You talk also about issue convincing ppl at Redhat to use Solid ?

> - L: There is a need to capture user data ( lightmode, language .. ), because of our ecosystem at Redhat and try to bring them together using GraphQL and I proposed Solid at a solution and the answer was "we already have system that can do what Solid can do ( auth etc...)" . I try to convice them that Solid is a better option. In there perspective if user don't request it, no need to implement it.

Reviewer: ```suggestion
- L: There is a need to capture user data ( lightmode, language .. ), because of our ecosystem at RedHat and efforts to bring them together using GraphQL. I proposed Solid at a solution, and the answer was "we already have systems that can do what Solid can do ( auth, etc...)" . I try to convince them that Solid is a better option. In their perspective, "if users don't request it, no need to implement it."
```

> - Jesse: At the ODI we are working on education and training project. How do you convice SME and org to use Solid. For SME, a good argument in favor of using Solid is they don't have to worry about the compliance issue . Would like to work with Luke to work on this issue ( convincing to use Solid)

Reviewer: ```suggestion
- Jesse: At the ODI we are working on education and training projects. How do you convince SME and org to use Solid? For SME, a good argument in favor of using Solid is they don't have to worry about the compliance issue. Would like to work with Luke on this issue (convincing to use Solid)
```

- WP : With business, is more simple if they don't have to manage the data. [...]

> - WP: In the economics of decentralised data and trading data the wallet is a great solution for that.

Reviewer: ```suggestion
- WP: In the economics of decentralised data and trading data, the wallet is a great solution.
```

> - Luke: A big issue is that ontology internally is all over the place, ppl have hard time to believe they can be one unique source of truth coming from the user

Reviewer: ```suggestion
- Luke: A big issue is that internal ontologies are all over the place. Ppl have a hard time believing there can be one unique source of truth coming from the user.
```

> - WP: will be great if redhat provide some infrastructure, as it is currently very embrionic

Reviewer: ```suggestion
- WP: will be great if redhat provide some infrastructure, as it is currently very embryonic
```

- JZ: Another important part of Solid is interoperability
- Tom Byrd: Another good selling point: [...]
- Marc Haddle: When it come to patient data: way better if have it in one place. Can be challenging to maintain the link of user data ( because of de-auth ). As long as data is hosted on someone else's server, you don't have control.
- Luke: Can be interesting for user centric AI-agent . ( Ai is currently a good selling point)

> - WP: AI has been a good selling point in our side. If redhat really want to differentiate itself from others, this can be a good point.

Reviewer: ```suggestion
- WP: AI has been a good selling point on our side. If RedHat really wants to differentiate itself from others, this can be a good point.
```

- J: One our side onthology has been a good branding, as it create AI ready data. Is RH working on onthology ?
- L: Low on the priority. The improvement from the onthology is obscure because the functionnality currently works. GDPR is not easy but working. I try to make a point on working on a small thing where Solid would work well.
- JZ: with redhat you get apache, why don't you get Solid too? wouldn't cost much for RedHat, and users could start playing with it.
- L: Yes and that's why we talked about keycloak, this is what we use by default.
- J: Pretty sure Inrupt makes Solid work with Keycloak but closed source
- FT: Would be very valuable to have this work. Be great to convice Tim to release this code

### Announcements, current topics relevant for the Group (please add your points!)

- JZ: there is a number of things discussed at the ODI advisory commitee:

#### use cases

- ODI is currently deciding which use cases to support.

> - J: clarification: we are not going to directly fund those use cases because of a limitation of the grant. We are building a framework to see which use cases should get support in priority. Call for action is to put Practitionners efforts into the Catalogue. Please reach to us if you need help or have any questions.

Reviewer: ```suggestion
- J: clarification: we are not going to directly fund use cases because of a limitation of the grant. We are building a framework to see which use cases should get support in priority. Call for action is to put Practitioners' efforts into the Catalogue. Please reach out to us if you need help or have any questions.
```

#### solid-contrib administration

- JZ: solid-contrib repository is currently in control of the Solid Team, the ODI would like to get admin instead of the Solid Team. The existing repos would remain in control of ppl who created them. ODI would not have the right to act on the PR except if not in the scope of the code of conduct. WDYT?
- J: proposed text for the Readme:

> ```
> Proposed text:
> 
> solid-contrib is an open space to work on software related to Solid. This GitHub organisation is administered by the Open Data Institute (ODI) - who are the Solid project stewards.
> 
> Individual repositories may have a set of owners from the community, if a repository does not have an owner - the ODI will take on ownership responsibility.
> 
> Owners of a repository have full responsibility and autonomy - the ODI will act as if it were an external contributor, the repository owner has final say in which PRs are merged. The only exception is if contents of the repository violate the Code of Conduct - in which case the ODI Ombuds <
> solid-ombuds@theodi.org
> > will moderate content.
> 
> If you wish to create a new repository, transfer a repository, or claim ownership of an existing repository contact,
> solid@theodi.org
> ```

Reviewer: ```suggestion
  <blockquote>
  Proposed text:

  solid-contrib is an open space to work on software related to Solid. This GitHub organisation is administered by the Open Data Institute (ODI) — who are the Solid project stewards.

  Individual repositories may have a set of owners from the community. If a repository does not have an owner, the ODI will take on ownership responsibility.

  Owners of a repository have full responsibility and autonomy — the ODI will act as if it were an external contributor. The repository owner has final say in which PRs are merged. The only exception would be if contents of the repository violate the Code of Conduct, in which case the ODI Ombuds <mailto:solid-ombuds@theodi.org> will moderate content.

  If you wish to create a new repository, transfer a repository, or claim ownership of an existing repository, contact <mailto:solid@theodi.org>
  </blockquote>
```

- J: I aimed to PR this next week, and have another week for the community to review and comment on the PR

> - Michal: would like to understand what is the purpose of replacing the Solid Team by the ODI? Also, has it been historically part of the Solid project or for the community?

Reviewer: ```suggestion
- Michal: I would like to understand the purpose of replacing the Solid Team with the ODI? Also, has it been historically part of the Solid project or for the community?
```
